qiskit/visualization/timeline/generators.py

```diff
@@ -129,17 +129,15 @@ def gen_sched_gate(
     except AttributeError:
         label = "n/a"
 
-    with warnings.catch_warnings():
-        warnings.simplefilter("ignore")
-        meta = {
-            "name": gate.operand.name,
-            "label": label,
-            "bits": ", ".join([bit.register.name for bit in gate.bits]),
```

Comment:
this metadata seems to be unused for the drawer (in particular, the `bits` element). If the drawer needs to access `bits`, it should also have access to the circuit context.

Reply:
Maybe @nkanazawa1989 can confirm or challenge this assumption

Reply:
Yes. This is not used for the current mpl drawer. My intention was using this information for interactive drawer (such as bokeh or plotly) so that developer can investigate the instruction details shown in a hover window. Such tool is more user-friendly especially when we visualize a deep circuit (they allow horizontal scroll). Unfortunately I didn't have time to actually implement the code.